pkg/tests/tests.go

```diff
@@ -56,3 +53,27 @@ var tests = []helpers.TestOptions{
 		Handler:  handlers.TestStaticEndpoint,
 	},
 }
+
+func accessReviewTest() helpers.TestOptions {
+	t := helpers.TestOptions{
+		TestName: "access-review",
+		Path:     "/api/authorizations/v1/access_review",
+		Method:   http.MethodPost,
+		Handler:  handlers.TestStaticEndpoint,
+	}
+	resourceReviewReq, err := authv1.NewAccessReviewRequest().
+		AccountUsername("rhn-support-tiwillia").
```

Comment:
can you please make this a `const` - so that if we ever need to share his username among tests any changes are only done to that single `const`.